### DISCUSSION TITLE
Resource is limited - try again in 4 hours (more than 100, code: "api-deployments-free-per-day")

### DISCUSSION THREAD

**Discussion by terygasmen** [oleeskild/obsidian-digital-garden#327] [Q&A]
Earlier today, I have this error sent to my email when publishing the changes for my Digital Garden. It says:

`Resource is limited - try again in 4 hours (more than 100, code: "api-deployments-free-per-day").`

I'm not sure if anyone else have this issue. Can anybody help? Does this mean I have to pay for vercel to keep on publishing the changes to my digital garden?

**Comment by shinanqu** [comment]
I think you can make 100 changes per day. If you often hit this limit, try to publish your changes less often. Publish only when you are sure the work is finished.

Also here is a workaround:
https://github.com/oleeskild/digitalgarden/issues/170

**Reply by gilipsy** [reply]
This is a limitation of the free version. I use Pro and there is no limit. If you want to stay within the limit, make fewer postings and changes.

**Comment by oleeskild** [comment]
I've finally gotten around to implement a fix for this, where multiple files will all be part of one single commit. So instead of having one commit per file, the plugin now makes a maximum of two commits for each publish: one for the changed/new files, and another one for deleted files. 

I've released it as a Beta version for now, that can be installed via BRAT. Any feedback, either here or in the [Discord](https://discord.gg/UsPH74nEVS) is much appreciated 🙏 

[Release Notes](https://github.com/oleeskild/obsidian-digital-garden/releases/tag/2.57.0)

**Comment by Shell1010** [comment]
A fix for this issue that I've accomplished is creating a new branch named "deployment" and pushing my changes to that branch whenever I want to push to production. I change the branch in vercel settings from main to deployment. I also updated my `vercel.json` so that any updated to the main branch will not create a deployment.
```json
{
  "git": {
    "deploymentEnabled": {
    "main": false
    }
  },
  "outputDirectory": "dist",
  "installCommand": "npm install",
  "buildCommand": "npm run build",
  "devCommand": "npm run start",
  "routes": [
    { "handle": "filesystem" },
    { "src": "/(.*)", "status": 404, "dest": "/404" }
  ]
}
```
